.github/workflows/mutest-issue-generate.yml

```diff
@@ -25,7 +24,7 @@ jobs:
       - name: Execute mutation test format script
         id: mutest-formatted
         run: |
-          cat mutation_test_result.txt | grep -Ev "PASS" | grep -Ev "SKIP" | tee mutation_test_result.txt
+          grep -Ev "PASS|SKIP" mutation_test_result.txt > mutation_test_result_temp.txt && mv mutation_test_result_temp.txt mutation_test_result.txt
```

Comment:
Note to reviewer: when running the previous code locally to clean up the results, the file would just be deleted. This is because we were attempting to write to the file while it was open, and resulted in lost data. This method is better so it can be run in both CI and locally without losing the file.